eessi/testsuite/tests/apps/openfoam/openfoam.py

```diff
@@ -164,3 +176,122 @@ def assert_sanity(self):
             self.assert_completion(),
             self.assert_convergence(),
         ])
+
+
+@rfm.simple_test
+class EESSI_OPENFOAM_LID_DRIVEN_CAVITY_1M(rfm.RunOnlyRegressionTest, EESSI_Mixin):
```

Comment:
you are repeating a lot of code from the 8M class to the 1M class, so this looks like an ideal case for centralizing those common things in a base class that both derive from.
are you up for that? i can help with this if you wnat.